test/database/ConnectionManagerTest.java

```diff
@@ -0,0 +1,22 @@
+package database;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+
+import org.junit.jupiter.api.Test;
+
+import config.Setting;
+import junit.framework.Assert;
+
+public class ConnectionManagerTest {
+	
+	private ConnectionManager connectionManager = new ConnectionManager();
+
+	@Test
+	public void testGetConnection() throws SQLException {
+		Connection connection = connectionManager.restoreConnection();
```

Comment:
The method, `restoreConnection()` does not exist anymore, please update.

Reply:
Please close the connection after the Assert on line 18th, and then make an Assert.true to evaluate the connection was effectively closed.